container/src/LuigiContainer.svelte

```diff
@@ -216,12 +135,29 @@
   onDestroy(async () => {});
 </script>
 
-<main
-  bind:this={mainComponent}
-  class={webcomponent ? undefined : 'lui-isolated'}
->
+<svelte:options
+  customElement={{ tag: null, props: { viewurl: { type: 'String', reflect: false, attribute: 'viewurl' }, deferInit: { type: 'Boolean', attribute: 'defer-init' }, noShadow: { type: 'Boolean', attribute: 'no-shadow' }, context: { type: 'String', reflect: false, attribute: 'context' }, label: { type: 'String', reflect: false, attribute: 'label' }, webcomponent: { type: 'String', reflect: false, attribute: 'webcomponent' }, locale: { type: 'String', reflect: false, attribute: 'locale' }, theme: { type: 'String', reflect: false, attribute: 'theme' }, activeFeatureToggleList: { type: 'Array', reflect: false, attribute: 'active-feature-toggle-list' }, skipInitCheck: { type: 'Boolean', reflect: false, attribute: 'skip-init-check' }, nodeParams: { type: 'Object', reflect: false, attribute: 'node-params' }, userSettings: { type: 'Object', reflect: false, attribute: 'user-settings' }, anchor: { type: 'String', reflect: false, attribute: 'anchor' }, searchParams: { type: 'Object', reflect: false, attribute: 'search-params' }, pathParams: { type: 'Object', reflect: false, attribute: 'path-params' }, clientPermissions: { type: 'Object', reflect: false, attribute: 'client-permissions' }, dirtyStatus: { type: 'Boolean', reflect: false, attribute: 'dirty-status' }, hasBack: { type: 'Boolean', reflect: false, attribute: 'has-back' }, documentTitle: { type: 'String', reflect: false, attribute: 'document-title' }, allowRules: { type: 'Array', reflect: false, attribute: 'allow-rules' }, sandboxRules: { type: 'Array', reflect: false, attribute: 'sandbox-rules' } }, extend: customElementConstructor => {
```

Comment:
Was this format done for a reason in one line 🤔

Reply:
ok so I see that the prettier for svelte moved the whole section here automatically. and kept the think in a one line

Reply:
I think its currently present and being used in v1 as per our [config](https://github.com/SAP/luigi/blob/main/prettier_config.json#L31), just with less sort filters. Newer versions I think [include the options ](https://github.com/sveltejs/prettier-plugin-svelte/blob/master/src/options.ts).

I think order though is not so important, but if we can with a small config change fix the one-liner would be easier to read.

Reply:
I decided to exclude files with 'svelte:options' from being pretiffied - this fix is already commited to this PR
FYI @JohannesDoberer